wifiphisher/pywifiphisher.py

```diff
@@ -136,6 +142,14 @@ def check_args(args):
             W +
             '] Pre-shared key must be between 8 and 63 printable characters.')
 
+    if args.handshake_capture and not os.path.isfile(
+            args.handshake_capture):
+        sys.exit('[' +
+                 R +
+                 '-' +
+                 W +
+                 '] handshake capture does not exist.')
+
```

Comment:
After some thinking, I think we can do the check here, but at that time we don't have the target AP's `BSSID` so I can only check if there is any `fourway-handshake` in the capture.
So for this reason the `_is_captured` flag in extension module should be preserved.